### DISCUSSION TITLE
What versions of scikit-learn are supported by 2024.1.X?

### DISCUSSION THREAD

**Discussion by carlos-rpg** [intel/scikit-learn-intelex#1760] [Q&A] [answered]
Hello,

The [system requirements](https://intel.github.io/scikit-learn-intelex/latest/system-requirements.html) state that:
> The latest release of scikit-learn-intelex-2023.1.X supports scikit-learn 0.24.X, 1.0.X, 1.1.X and 1.2.X

But intelex is already on version 2024.1.X. What are the current supported versions?

thanks in advance

**Comment by napetrov** [comment, marked as answer]
@carlos-rpg  - it's 1.x + currently - current state is reflected in README.md - https://github.com/intel/scikit-learn-intelex
But previous versions might still work, although they are not tested 
@aepanchi  - looks we need to deploy updated docs?

**Reply by carlos-rpg** [reply]
Thank you! By "currently" you mean also 1.4? On the README.md it says from 1.0 to 1.3.

**Reply by napetrov** [reply]
It's being updated regularly but in general target is to support  new scikit-learn version at release time or ight after it

**Reply by carlos-rpg** [reply]
I see that README.md now shows compatibility up to 1.6, but the documentation still says up to 1.3: https://uxlfoundation.github.io/scikit-learn-intelex/2025.2/quick-start.html#software-requirements
I should consider the question answered anyway, thanks!

**Reply by napetrov** [reply]
Yes, it's fixed in sources but not yet deployed to docs.  @yuejiaointel - can we get last version deployed?
